src/services/codefixes/importFixes.ts

```diff
@@ -0,0 +1,291 @@
+﻿/* @internal */
+namespace ts.codefix {
+    const nodeModulesDir = "node_modules";
+
+    registerCodeFix({
+        errorCodes: [Diagnostics.Cannot_find_name_0.code],
+        getCodeActions: (context: CodeFixContext) => {
+            const sourceFile = context.sourceFile;
+            const checker = context.program.getTypeChecker();
+            const allFiles = context.program.getSourceFiles();
+            const useCaseSensitiveFileNames = context.host.useCaseSensitiveFileNames ? context.host.useCaseSensitiveFileNames() : false;
+
+            const token = getTokenAtPosition(sourceFile, context.span.start);
+            const name = token.getText();
+            const allActions: CodeAction[] = [];
+
+            // Get existing ImportDeclarations from the source file
+            const imports: ImportDeclaration[] = [];
+            if (sourceFile.statements) {
+                for (const statement of sourceFile.statements) {
+                    if (statement.kind === SyntaxKind.ImportDeclaration) {
+                        imports.push(<ImportDeclaration>statement);

```

Comment:
The closest thing I can find is the `sourceFIle.Imports`, however I don't know how to get the import declarations from the module names. Maybe I can call `getSymbolAtLocation`? Or do you know there are better ways.